source/settings/plugin/acl-plugin.rst

```diff
@@ -0,0 +1,178 @@
+.. _plugin-acl:
+
+==========
+acl plugin
+==========
+
+.. seealso:: See :ref:`acl` for a plugin overview.
+
+Settings
+^^^^^^^^
+
+.. _plugin-acl-setting_acl:
+
+``acl``
+-------
+
+- Default: <empty>
+- Values:  :ref:`string`
+
+The ACL driver to use. This setting is **REQUIRED** - if empty, the acl plugin
+is disabled.
+
+The format is:
+
+.. code-block:: none
+
+  backend[:option[:option...]]
+
+Currently, there is a single backend available: ``vfile``. This backend
+supports two ways of defining the ACL configuration:
+
+  * *global*: ACL rules are applied to all users.
+  * *per-mailbox*: Each mailbox has separate ACL rules. They are stored in a
+    ``dovecot-acl`` file in each mailbox's index (or ``CONTROL``) directory.
```

Comment:
Not index. Maybe `in each mailbox (or CONTROL) directory.`

Reply:
Ugh. Thought I did that.  Fixed for real now.